apps/music/views/home/view.js

```diff
@@ -19,7 +19,8 @@ var HomeView = View.extend(function HomeView() {
 
   this.searchResults.addEventListener('close', () => {
     this.client.method('searchClose');
-    document.body.removeAttribute('data-search');
+    document.body.dataset.search = false;
+    window.scrollTo(0, this.searchBox.HEIGHT);
```

Comment:
It might be worth adding a note here that we eventually want to turn this into a smooth scroll.